### PR TITLE
#162948985 Setup and deploy base app to heroku

### PR DESCRIPTION
**Pull request by charitymarani**
**What does this PR do?**
Sets up and deploys a base app to Heroku

**Description of Task to be completed?**
Created a home route for a welcome message
Created a pipeline on Heroku with review apps, staging, and production
Configured app.json file

**How should this be manually tested?**
Here are the app URLs:

**Reviewapp**-https://ah-fulldeck-staging-pr-7.herokuapp.com/
**Staging**-https://ah-fulldeck-staging.herokuapp.com/
**Production**-https://ah-fulldeck.herokuapp.com/

**Related PT story**
[162948985](https://www.pivotaltracker.com/story/show/162948985)

### CODE REVIEW

**Review by Hesbon5600 — COMMENTED** [review]
Great work on adding a custom welcome message

File: authors/apps/base/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.home, name='home'),
+]
```
Comment (Nicanor008) [inline authors/apps/base/urls.py]: Add a new blank line at the end of file, that's as per pep8 style guide @charitymarani

File: authors/settings.py [hunk 2]
```diff
@@ -144,3 +146,4 @@
         # 'authors.apps.authentication.backends.JWTAuthentication',
     ),
 }
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```
Comment (eduhmik) [inline authors/settings.py]: Consider adding this line of code which helps to access files in the static folder like the CSS
STATIC_ROOT = os.path.join(BASE_DIR, 'static/')

Comment (charitymarani) [reply authors/settings.py]: @eduhmik I have handled this using django_heroku

File: authors/apps/base/urls.py (hunk 1 above)
Comment (charitymarani) [reply authors/apps/base/urls.py]: @Nicanor008 working on it

**Comment by kzyangiro** [comment]
@charitymarani The naming is not quite okey. I would prefer a better name/url i.e https://ah-fulldeck.herokuapp.com/ without iondicating the PR and PR id

**Comment by charitymarani** [comment]
> @charitymarani The naming is not quite okey. I would prefer a better name/url i.e https://ah-fulldeck.herokuapp.com/ without iondicating the PR and PR id

@kzyGit I created a pipeline, so the app https://ah-fulldeck-staging-pr-7.herokuapp.com/ is a review app and not the actual app. The staging app is **ah-fulldeck-staging**  and production app is called **ah-fulldeck**

**Comment by kzyangiro** [comment]
> > @charitymarani The naming is not quite okey. I would prefer a better name/url i.e https://ah-fulldeck.herokuapp.com/ without iondicating the PR and PR id
> 
> @kzyGit I created a pipeline, so the app https://ah-fulldeck-staging-pr-7.herokuapp.com/ is a review app and not the actual app. The staging app is **ah-fulldeck-staging** and production app is called **ah-fulldeck**

@charitymarani I need you to add the full detail on the PR indicating both staging and production URLs aswell.

**Review by kzyangiro — APPROVED** [review]

**Pull request merged**